src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import core.basesyntax.suppliers.FigureSupplier;
+import core.basesyntax.templates.Drawable;
+import core.basesyntax.templates.Figure;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = i < 3 ? FigureSupplier.getRandomFigure()
+                    : FigureSupplier.getDefaultFigure();

```

Comment:
**Don’t use static methods in your solution**

[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-static-methods-in-your-solution)